### ISSUE TITLE
Wit recognize ordinal number as a date

### ISSUE THREAD

**Issue by misysdclabs** [wit-ai/wit#294]
I have a new story in my app where I would like to recognize simple ordinal numbers like third, 4th, eleventh, etc. I try to use the built-in wit/ordinal entity for this purpose, which just doesn't want to fit. I got a red flag for this:

<img width="550" alt="screen shot 2016-08-17 at 10 43 40" src="https://cloud.githubusercontent.com/assets/14309287/17730150/c885e8d8-6467-11e6-90fe-6be383b1e211.png">

When I try to teach it in the Understanding menu, Wit recognize my ordinals as wit/datetime, even after a dozen relearning phase:

<img width="616" alt="screen shot 2016-08-17 at 09 23 05" src="https://cloud.githubusercontent.com/assets/14309287/17730226/267ea2fe-6468-11e6-842f-5f1c845ce618.png">

Notes: I have another app where I started to use ordinal numbers earlier. That app still works perfectly with ordinals:

<img width="637" alt="screen shot 2016-08-17 at 10 51 27" src="https://cloud.githubusercontent.com/assets/14309287/17730347/a1a385c6-6468-11e6-8b0c-fb69eaa0a71b.png">

Please help.

**Comment by misysdclabs** [comment]
Update, I moved back to that app what I mentioned, where I have a story with working wit/ordinal recognizing. I changed a couple of things, removed some actions and responses to fit my current client script.

After the changes my stories went yellow and after a bit back to green. But, the whole app remained in yellow, forever:

<img width="362" alt="screen shot 2016-08-17 at 11 18 51" src="https://cloud.githubusercontent.com/assets/14309287/17731456/ac5c5cf4-646d-11e6-8d9d-7ab90df246f0.png">

Training status: ongoing.

When I start my client, my app works totally wrong, go into a wrong story, than calls a function which is  called in a third story, etc.

What happened here? Something went really wrong with this guys.
